src/deepsparse/server/README.md

```diff
@@ -0,0 +1,120 @@
+                                        0.0.0.0
+
+        --port INTEGER                  Bind to a socket with this port. Defaults to
+                                        5543.
+
+        --workers INTEGER               Use multiple worker processes. Defaults to
+                                        1.
+
+        --log_level                     [debug|info|warn|critical|fatal]
+                                        Bind to a socket with this port. Defaults to
+                                        info.
+
+        --config_file TEXT              Configuration file containing info on how to
+                                        serve the desired models.
+
+        --task TEXT                     The task the model_path is serving. For
+                                        example, one of: question_answering,
+                                        text_classification, token_classification.
+
+                                        Ignored if config file is supplied
+        --model_path TEXT               The path to a model.onnx file, a model
+                                        folder containing the model.onnx and
+                                        supporting files, or a SparseZoo model stub.
+                                        Ignored if config_file is supplied.
+
+        --batch_size INTEGER            The batch size to serve the model from
+                                        model_path with. Ignored if config_file is
+                                        supplied.
+
+        --help                          Show this message and exit.
+
+
+##### Single Model Inference
+
+Example CLI command for serving a single model:
+
+```bash
+deepsparse.server \
+    --task question_answering \
+    --model_path "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none"
+```
+
+##### Multiple Model Inference
+To serve multiple models you can easily build a `config.yaml` file. 
+In the sample YAML below, we are defining two BERT models to be served by the `deepsparse.server` for the question answering task:
+
+    models:
+    - task: question_answering
+        model_path: zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none
+        batch_size: 1
+        alias: question_answering/dense
+    - task: question_answering
+        model_path: zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant-aggressive_95
+        batch_size: 1
+        alias: question_answering/sparse_quantized
+
+After you finish building the `config.yaml` file, you can run the server with the config file path passed in the `--config_file` argument:
+```bash
+deepsparse.server --config_file config.yaml
+```
+
+💡 PRO-TIP 💡: When your server is running, you can always use the awesome swagger UI that's built into FastAPI to view your model's pipeline `POST` routes. All you need is to add `/docs` at the end of your host URL:
+
+    localhost:5543/docs
+
+![alt text](./img/swagger_ui.png)
```

Comment:
Can you connect serving and client usage a bit more? The section of "Single Model Inference" gives the user little to go off of how to use it afterwards or that they should move onto client

Reply:
just realized the client example I included can only be used for multiple models because of the `alias`